src/Nox.Solution/Nox.Solution.csproj

```diff
@@ -24,6 +24,7 @@
 	<ItemGroup>
 		<PackageReference Include="FluentValidation" Version="11.7.1" />
 		<PackageReference Include="Humanizer.Core" Version="2.14.1" />
+		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.7.0" />
```

Comment:
do we need this reference?

Reply:
Using this library was the easiest way to check for reserved keywords. I'm having to do this since we now have the FormulaReturnType enum using protected keywords and they were not being properly generated in the sample projects.
We are already using this library in other projects, so at least it's not a new library added to the solution.